src/Knp/Menu/Util/MenuManipulator.php

```diff
@@ -265,6 +275,10 @@ public function getBreadcrumbsArray(ItemInterface $item, $subItem = null): array
         return $breadcrumbs;
     }
 
+    /**
+     * @return array<int, array<string, string|ItemInterface|null>>
```

Comment:
I would remove that one, keeping only the `@phpstan-return` and the `array` return type. `array<string, string|ItemInterface|null>` would create confusion as it would imply that any of the keys could be of the type `string|ItemInterface|null`.
PHPStan won't complain about an unspecified array type, as it is specified in `@phpstan-return`.